### ISSUE TITLE
opener_paths.xml" already exists!

### ISSUE THREAD

**Issue by fredlemieux** [pwlin/cordova-plugin-file-opener2#193]
I got the following error when trying to install cordova-plugin-file-opener2:

```
Error during processing of action! Attempting to revert...
Failed to install 'cordova-plugin-file-opener2': Error
    at copyNewFile (C:\Dev\Ionic\snaggingappkeith\platforms\android\cordova\lib\pluginHandlers.js:261:45)
    at install (C:\Dev\Ionic\snaggingappkeith\platforms\android\cordova\lib\pluginHandlers.js:51:17)
    at ActionStack.process (C:\Dev\Ionic\snaggingappkeith\platforms\android\cordova\node_modules\cordova-common\src\ActionStack.js:56:25)
    at PluginManager.doOperation (C:\Dev\Ionic\snaggingappkeith\platforms\android\cordova\node_modules\cordova-common\src\PluginManager.js:114:20)
    at PluginManager.addPlugin (C:\Dev\Ionic\snaggingappkeith\platforms\android\cordova\node_modules\cordova-common\src\PluginManager.js:144:17)
    at C:\Dev\Ionic\snaggingappkeith\platforms\android\cordova\Api.js:247:74
    at _fulfilled (C:\Dev\Ionic\snaggingappkeith\platforms\android\cordova\node_modules\q\q.js:854:54)
    at self.promiseDispatch.done (C:\Dev\Ionic\snaggingappkeith\platforms\android\cordova\node_modules\q\q.js:883:30)
    at Promise.promise.promiseDispatch (C:\Dev\Ionic\snaggingappkeith\platforms\android\cordova\node_modules\q\q.js:816:13)
    at C:\Dev\Ionic\snaggingappkeith\platforms\android\cordova\node_modules\q\q.js:570:49
(node:1284) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Uh oh!
"C:\Dev\Ionic\snaggingappkeith\platforms\android\app\src\main\res\xml\opener_paths.xml" already exists!
(node:1284) DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.
js process with a non-zero exit code.
```

I think this is due to having "cordova-plugin-file" already installed which I assume also creates this "opener_paths.xml" file? I managed to get round this issue by removing both plugins and installing the "cordova-plugin-file-opener2" first then installing the cordova-plugin-file and it now seems to work fine.

This issue seems to have only started happening since a recent update to android version 7 (it's messed up a few other plugins too).  

I'm not sure if this classifies as an issue, but I thought I'd share this in case anyone else has the same problem.

**Comment by ajayadglobal360** [comment]
Run this ionic cordova plugin add cordova-plugin-file-opener2 --save --force

**Comment by shnist** [comment]
hi @fredlemieux, thanks for raising this issue. After doing some investigation it seems the issue was related to the android support v4 version issue: 

https://forum.ionicframework.com/t/cordova-plugin-file-opener2-installing-and-build-error-android/124304

This has been fixed since 2.1.0, so closing.
